packages/synapse-react-client/src/components/EntityCitation/EntityCitation.tsx

```diff
@@ -0,0 +1,205 @@
+      setCitationAnchorEl(event.currentTarget)
+    } else {
+      // If both project and entity doi exist, show dropdown
+      setAnchorEl(event.currentTarget)
+    }
+  }
+
+  const handleSelectCitation = (doi: string) => {
+    setSelectedDoi(doi)
+    setAnchorEl(null)
+    setCitationAnchorEl(anchorEl)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const popover = (
+    <CitationPopoverContent
+      doi={selectedDoi}
+      anchorEl={citationAnchorEl}
+      open={!!citationAnchorEl && !!selectedDoi}
+      id={id}
+      onClose={() => {
+        setCitationAnchorEl(null)
+        setSelectedDoi(undefined)
+      }}
+    />
+  )
+
+  const dropdown = (
+    <Popover
+      open={open}
+      anchorEl={anchorEl}
+      onClose={handleClose}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'left',
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'left',
+      }}
+      PaperProps={{
+        sx: {
+          padding: '10px 0',
+          borderRadius: '0px',
+          boxShadow: '0px 8px 24px 0px rgba(53, 58, 63, 0.15)',
+        },
+      }}
+    >
+      <MenuItem
+        value={projectDoiAssociation?.doiUri}
+        onClick={() => handleSelectCitation(projectDoiAssociation?.doiUri!)}
+      >
+        Cite this Project
+      </MenuItem>
+      <MenuItem
+        value={entityDoiAssociation?.doiUri}
+        onClick={() => handleSelectCitation(entityDoiAssociation?.doiUri!)}
+      >
+        Cite only this page
+      </MenuItem>
+    </Popover>
+  )
+
+  return (
+    <>
+      {(projectDoiAssociation || entityDoiAssociation) && (
+        <>
+          <Button
+            onClick={handleButtonClick}
+            sx={{
+              '& .MuiButton-endIcon': {
+                margin: 0,
+              },
+              borderColor: '#FFF !important',
+              color: '#FFF !important',
+              borderRadius: '3px',
+              padding: '6px 16px',
+              '&:hover': {
+                backgroundColor: '#304b66',
+              },
+              ...(isProjectAndEntityDoi && { paddingRight: '0px' }),
+              width: { xs: '100%', sm: 'initial' },
+            }}
+            variant="outlined"
+            startIcon={<DoubleQuotes width={18} height={18} />}
+            endIcon={
+              isProjectAndEntityDoi && (
+                <Box
+                  sx={{
+                    borderLeft: '1px solid white',
+                    marginTop: '-8px',
+                    marginBottom: '-8px',
+                    display: 'flex',
+                    marginLeft: '16px',
+                    padding: '5px',
+                  }}
+                >
+                  <ExpandMoreIcon
+                    className="user-expand"
+                    width={18}
+                    height={18}
+                  />
+                </Box>
+              )
+            }
+          >
```

Comment:
Maybe instead of negative margin icon styling, could we use the [`DropdownSelect`](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/48ba269795ecd5875af9e1cc816c6f771d405ac0/packages/synapse-react-client/src/components/DropdownSelect/DropdownSelect.tsx#L31) component when we have both DOIs?